test cases/common/153 wrap file should not failed/meson.build

```diff
@@ -14,3 +14,8 @@ executable('grabprog2', files('src/subprojects/foo/prog2.c'))
 subdir('src')
 
 subproject('patchdir')
+
+if find_program('patch', required : false).found() or find_program('git', required : false).found()
+  exe = subproject('patchfile').get_variable('foo_exe')
```

Comment:
Feels like this should maybe be a MESON_SKIP_TEST?

Reply:
You mean, we should skip this whole test case when there's no `patch` / `git`? OK, done.